src/shared/containers/ResourceEditor.tsx

```diff
@@ -54,12 +74,8 @@ const ResourceEditorContainer: React.FunctionComponent<{
           error: null,
           busy: true,
         });
-        const newResource = expanded
-          ? await nexus.Resource.get(orgLabel, projectLabel, resourceId, {
-              rev,
-              format: 'expanded',
-            })
-          : await nexus.Resource.getSource(orgLabel, projectLabel, resourceId);
+
+        const  newResource =  await getNewResource();
```

Comment:
I think there are extra spaces here after `const` and `await` :)

Reply:
Ran prettier. They should be good now.